extensions/ipynb/src/serializers.ts

```diff
@@ -30,12 +30,35 @@ export function createJupyterCellFromNotebookCell(
 	return cell;
 }
 
+
+/**
+ * Sort the JSON to minimize unnecessary SCM changes.
+ * Jupyter notbeooks/labs sorts the JSON keys in alphabetical order.
+ * https://github.com/microsoft/vscode-python/issues/13155
+ */
+export function sortObjectPropertiesRecursively(obj: any): any {
```

Comment:
This was missed when moving the serializer into vscode, adding this back.
I think at one point we relialized this was missing, can't remember. but added comments with the link to the original issue so we have some history.

Reply:
I think there is an issue on the vscode side for it but I thought we figured out that it wasn't actually being done in the other serializer at the time we moved it